### PR TITLE
+TCP: Fix spellings

### PR DESCRIPTION
**Pull request by AniruddhaKanhere**
Description
-----------
This PR adds multiple spelling corrections in files related to the TCP source code.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: FreeRTOS-Plus/Test/CBMC/proofs/CBMCStubLibrary/tasksStubs.c [hunk 1]
```diff
@@ -23,7 +23,7 @@ BaseType_t xTaskGetSchedulerState( void )
 	return xState;
 }
 
-/* This function is another method apart from overwritting the defines to init the max
+/* This function is another method apart from over-writting the defines to init the max
```
Comment (divekarshubham) [inline FreeRTOS-Plus/Test/CBMC/proofs/CBMCStubLibrary/tasksStubs.c:26]: should be overwriting

**Review by divekarshubham — CHANGES_REQUESTED** [review]
Just one spell change

**Comment by AniruddhaKanhere** [comment]
> Just one spell change

Thanks a lot Shubham! Nice catch.

**Review by divekarshubham — APPROVED** [review]

File: FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DHCP.c [hunk 2]
```diff
@@ -627,7 +627,7 @@ static void prvInitialiseDHCP( void )
 	}
 	else
 	{
-		/* There was a problem with the randomiser. */
+		/* There was a problem with the randomizer. */
```
Comment (gkwicker) [inline FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DHCP.c:630]: This is changing from English to American spelling – our dictionary is probably using American spelling.

File: FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_TCP_IP.c [hunk 3]
```diff
@@ -1020,7 +1020,7 @@ uint32_t ulInitialSequenceNumber = 0;
 	case eARPCacheMiss:		/* An ARP table lookup did not find a valid entry. */
 	case eCantSendPacket:	/* There is no IP address, or an ARP is still in progress. */
 	default:
-		/* Count the number of times it couldn't find the ARP address. */
+		/* Count the number of times it could not find the ARP address. */
```
Comment (gkwicker) [inline FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_TCP_IP.c:1023]: This was fine before – the spell checker is not perfect and can't handle contractions.  You will see things like _wouldn_, _shouldn_, etc. in the lexicon.txt.

**Review by gkwicker — APPROVED** [review]

**Pull request merged**